src/main/java/com/capitalone/dashboard/logging/LoggingFilter.java

```diff
@@ -52,6 +52,8 @@
 @Order(1)
 public class LoggingFilter implements Filter {
 
+    public static final String IGNORE_TEST_RESULTS_REQUESTS = "/api/quality/test-result";
```

Comment:
Can we externalize this as we might have more apis that need to be added to the exclusion list. You could maintain a list of exclusion uris that can be overridden by properties.

Reply:
put it into the properties file